examples/output_voltage_measurement/measurement.py

```diff
@@ -106,118 +87,100 @@ def measure(
-    with session_management_client.reserve_sessions(
-        context=measurement_service.context.pin_map_context,
-        pin_or_relay_names=[input_pin, output_pin],
-    ) as reservation:
-        grpc_device_channel = get_grpc_device_channel(
-            measurement_service, nidcpower, service_options
-        )
-        source_session_info = _get_session_info_for_pin(reservation.session_info, input_pin)
-        measure_session_info = _get_session_info_for_pin(reservation.session_info, output_pin)
-        with _nidcpower_helpers.create_session(
-            source_session_info, service_options.use_simulation, grpc_device_channel
-        ) as source_session, _visa_helpers.create_session(
-            measure_session_info.resource_name,
-            use_simulation=service_options.use_simulation,
-        ) as measure_session:
-            cancellation_event = threading.Event()
-            measurement_service.context.add_cancel_callback(cancellation_event.set)
-
-            assert isinstance(measure_session, pyvisa.resources.MessageBasedResource)
-
-            log_instrument_id(measure_session)
-
-            # When this measurement is called from outside of TestStand (session_exists == False),
-            # reset the instrument to a known state. In TestStand, ProcessSetup resets the
-            # instrument.
-            if not measure_session_info.session_exists:
-                reset_instrument(measure_session)
-
-            channels = source_session.channels[source_session_info.channel_list]
-
-            channels.source_mode = nidcpower.SourceMode.SINGLE_POINT
-            channels.output_function = nidcpower.OutputFunction.DC_VOLTAGE
-            channels.current_limit = current_limit
-            channels.voltage_level_range = voltage_level_range
-            channels.current_limit_range = current_limit_range
-            channels.source_delay = hightime.timedelta(seconds=source_delay)
-            channels.voltage_level = voltage_level
-
-            # Configure NI-VISA DMM
-            function_enum = FUNCTION_TO_VALUE[measurement_type]
-            resolution_value = RESOLUTION_DIGITS_TO_VALUE[str(resolution_digits)]
-            measure_session.write("CONF:%s %.g,%.g" % (function_enum, range, resolution_value))
-            check_instrument_error(measure_session)
-
-            with channels.initiate():
-                _wait_for_source_complete_event(measurement_service, channels, cancellation_event)
-
-            response = measure_session.query("READ?")
-            check_instrument_error(measure_session)
-            measured_value = float(response)
-
-            source_session = None  # Don't abort after this point
+    cancellation_event = threading.Event()
+    measurement_service.context.add_cancel_callback(cancellation_event.set)
+
+    with measurement_service.context.reserve_sessions([input_pin, output_pin]) as reservation:
+        with reservation.create_nidcpower_session() as _, reservation.create_session(
```

Comment:
This example doesn't need the value yielded by the context manager.

@dixonjoel @pbirkhol-ni Which is less confusing?
- `with reservation.create_nidcpower_session() as _:`
- `with reservation.create_nidcpower_session():`

Reply:
I prefer the latter. I think it's clear that it's just for the purpose of the session lifetime.

Reply:
ok, I deleted `as _`